modules/running-insights-operator-gather.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: PROCEDURE
+[id="running-insights-operator-gather_{context}"]
+= Running an Insights Operator gather operation
+
+You can run an Insights Operator gather operation manually. If you have previously disabled Insights Operator gather operations in your cluster, this procedure will override that. 
+
+:FeatureName: The `InsightsDataGather` custom resource 
+include::snippets/technology-preview.adoc[] 
+
+[NOTE]
+====
+Enabling Technology Preview in your cluster will cause Insights Operator to run gather operations in individual pods. This is part of the Technology Preview feature set for the Insights Operator and supports the new data gathering features.
+====
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` role.
+
+.Procedure
+
+. Navigate to *Administration* -> *CustomResourceDefinitions*.
+. On the *CustomResourceDefinitions* page, use the *Search by name* field to find the *DataGather* resource definition and click it.
+. On the *CustomResourceDefinition details* page, click the *Instances* tab.
+. Click *Create DataGather*.
+. To create a new `DataGather` operation, edit the configuration file:
++
+[source,yaml]
+----
+apiVersion: insights.openshift.io/v1alpha1
+kind: DataGather
+metadata:
+  name: <your_data_gather> <1>
+spec:
+ gatherers: <2>
+   - name: workloads 
+     state: Disabled
+----
++
+--
+<1> Replace the `<your_data_gather>` with a unique name for your gather operation. 
```

Comment:
Perhaps I would also add the info that it must not be prefixed with `periodic-gathering`

Reply:
Done